compiler/quilt/test/test_import.py

```diff
@@ -430,3 +430,39 @@ def test_team_set_non_node_attr(self):
         df = pd.DataFrame(dict(a=[1, 2, 3]))
         with self.assertRaises(AttributeError):
             package4.newdf = df
+
+    def test_datanode_asa(self):
+
+        def test_lambda(node, hashes):
```

Comment:
Might be good to actually return something from the lambda, and check the `.csv(...)` return value.